src/locale/languages/nl.js

```diff
@@ -0,0 +1,311 @@
+      userData2:
+        'Dit betekent dat wij op geen enkele wijze de GitHub-gegevens van een gebruiker ooit kunnen inzien, gebruiken of delen. Als privégegevens ooit zichtbaar worden, zullen we het niet opnemen of bekijken. Als het per ongeluk wordt opgenomen, zullen wij het onmiddellijk verwijderen met veilige verwijderingsmethoden. Nogmaals, wij hebben authentificatie speciaal ingesteld zodat dit nooit gebeurt.',
+      analyticsInfoTitle: 'ANALYTICS INFORMATIE',
+      analyticsInfo1:
+        'We gebruiken op dit moment Google Analytics en iTunes App Analytics om ons te helpen met het meten van verkeer en gebruikstendensen voor het GitPoint. Deze tools verzamelen informatie die door uw apparaat wordt verzonden, inclusief apparaat- en platformversie, regio en verwijzer. Deze informatie kan niet worden gebruikt om een bepaalde individuele gebruiker te identificeren en er worden geen persoonlijke gegevens geëxtraheerd.',
+      analyticsInfo2:
+        'Als wij een ander platform van derden opnemen om problemen, fouten of meer analytische informatie te verzamelen, zullen wij ervoor zorgen dat de gebruikersgegevens geanonimiseerd en gecodeerd zijn.',
+      openSourceTitle: 'OPEN SOURCE',
+      openSource1:
+        'GitPoint is open source en de geschiedenis van de bijdragen aan het platform is altijd zichtbaar voor het publiek.',
+      openSource2:
+        'Bij elke bijdrage aan de app wordt er een code review uitgevoerd om te voorkomen dat iemand schadelijke codes toevoegd.',
+      contactTitle: 'CONTACT',
+      contact1:
+        'Bedankt voor het lezen van ons privacybeleid. Wij hopen dat u het leuk vindt om GitPoint te gebruiken net zoals wij het leuk vinden om het te bouwen.',
+      contact2:
+        'Als u vragen heeft over dit Privacybeleid of GitPoint in het algemeen, gelieve een probleem in de',
+      contactLink: 'GitPoint repository',
+    },
+  },
+  notifications: {
+    main: {
+      unread: 'ongelezen',
+      participating: 'deelnemend',
+      all: 'Alles',
+      unreadButton: 'Ongelezen',
+      participatingButton: 'Deelnemend',
+      allButton: 'Alles',
+      retrievingMessage: 'Notificaties ophalen',
+      noneMessage: 'Je hebt geen notificaties met dit type',
+    },
+  },
+  search: {
+    main: {
+      repositoryButton: 'Repositories',
+      userButton: 'Gebruikers',
+      searchingMessage: 'Zoeken naar {{query}}',
+      searchMessage: 'Zoeken naar {{type}}',
+      repository: 'repository',
+      user: 'gebruiker',
+    },
+  },
+  user: {
+    profile: {
+      userActions: 'Gebruiker Acties',
+      unfollow: 'Ontvolgen',
+      follow: 'Volgen',
+    },
+    repositoryList: {
+      title: 'Repositories',
+    },
+    followerList: {
+      title: 'Volgers',
+    },
+    followingList: {
+      title: 'Volgen',
+    },
+  },
```

Comment:
For #260 PR there is a [new phrase](https://github.com/gitpoint/git-point/pull/260/files#diff-f9a4a43ac6485b52db67745369cbcc2d) for translation, you can add it below now:
```js
followYou: {
 title: 'Follows you',
},
```